node/sifrr/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "@sifrr/server": "0.0.6",
+    "graphql": "^14.4.2",
```

Comment:
can you remove this constraint ?

Reply:
version or dependency?

Reply:
yeh, even use `*` for `graphql` or use a strict version contraint`

even better made this optional when using `sifrr` :stuck_out_tongue:

Reply:
I will, 😂 in next version

Reply:
then, I'll please pleased to review a `PR`, reving this dependency :heart: